src/WaCore.Sample/Startup.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using WaCore.Sample.Data;
+using Microsoft.EntityFrameworkCore;
+using WaCore.Sample.Data.Repositories;
+
+namespace WaCore.Sample
+{
+    public class Startup
+    {
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        // This method gets called by the runtime. Use this method to add services to the container.
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddMvc();
+
+            var connection = @"Server=(localdb)\mssqllocaldb;Database=WaCore.Sample;Trusted_Connection=True;";
```

Comment:
Take a look at InMemory DB option of EF (https://docs.microsoft.com/en-us/ef/core/miscellaneous/testing/in-memory) 
I've never used it by myself (so I don't know whether it's suitable here) but I was always willing to try.